docs/admin/admin-ui-setup/dashboard-analysis.md

```diff
@@ -2,95 +2,106 @@
 
 The Run:ai Administration User Interface provides a set of dashboards that help you monitor Clusters, Cluster Nodes, Projects, and Workloads. This document provides the key metrics to monitor, how to assess them as well as suggested actions.
 
+Dashboards are used by system administrators to analyze and diagnose issues that relate to:
```

Comment:
All this new section is related to the new Overview. So it should be under the overview dashboard in line 13

Reply:
Actually this is more a general description of all the dashboards and should conceptually hold true for all of them. If there are dashboards that are not used by Admins, meaning like a researcher or other then I can add those roles to sentence.